File: src/1450.number-of-students-doing-homework-at-a-given-time.1560/solution.ts
function busyStudent (startTime: number[], endTime: number[], queryTime: number): number {
    let result = 0;
    for (let i = 0; i < startTime.length; i++) {
        if (queryTime >= startTime[i] && queryTime <= endTime[i]) {
            result++;
        }
    }
    return result;
}

File: src/1450.number-of-students-doing-homework-at-a-given-time.1560/solution.java
class Solution {
    public int busyStudent(int[] startTime, int[] endTime, int queryTime) {
        int cnt = 0;
        for(int i=0;i<startTime.length;i++){
            if(startTime[i]<=queryTime && endTime[i]>=queryTime){
                cnt++;
            }
        }
        return cnt;
    }
}